Source/DafnyLanguageServer/Language/Symbols/FunctionSymbol.cs

```diff
@@ -8,7 +9,23 @@ public class FunctionSymbol : MemberSymbol, ILocalizableSymbol {
 
     public IList<VariableSymbol> Parameters { get; } = new List<VariableSymbol>();
 
-    public override IEnumerable<ISymbol> Children => Parameters;
+    /// <summary>
+    /// Gets the body of the function
+    /// </summary>
+    public ScopeSymbol? Body { get; set; }
+    public ScopeSymbol? ByMethodBody { get; set; }
+    public List<ScopeSymbol> Ens { get; set; }
+    public List<ScopeSymbol> Req { get; set; }
+    public List<ScopeSymbol> Reads { get; set; }
+    public List<ScopeSymbol> Decreases { get; set; }
+    public override IEnumerable<ISymbol> Children =>
+      AsEnumerable<ISymbol>(Body)
+        .Concat(AsEnumerable(ByMethodBody))
```

Comment:
Use `Append` to add single entities.

Reply:
`ByMethodBody` being nullable, we don't want `Append`, and I'm not aware of an `AppendIfNotNull`